examples/triton/docker-compose.yaml

```diff
@@ -0,0 +1,39 @@
+version: '2.1'
+# Cassandra demonstration of the Autopilot pattern
+
+services:
+  cassandra:
+    image: autopilotpattern/cassandra:latest
+    restart: always
+    mem_limit: 4g
+    network_mode: bridge
+    labels:
+      - triton.cns.services=cassandra
+    environment:
+      - CONSUL=cassandra-consul.svc.${TRITON_CNS_SEARCH_DOMAIN_PRIVATE}
+      - CASSANDRA_USER=cassandra
+      - CASSANDRA_PASSWORD=cassandra
+      - CASSANDRA_CLUSTER_NAME=Test Cluster
+      - CASSANDRA_KEYSPACES=demo
+      - CASSANDRA_TOPOLOGY={"demo":{"datacenter1":1}}
+    links:
+      - consul:consul
+
+  # Start with a single host which will bootstrap the cluster.
+  # In production we'll want to use an HA cluster.
+  consul:
+    image: autopilotpattern/consul:0.7.2-r0.8
```

Comment:
We've qualified Consul 1.0 support with the latest version of ContainerPilot and `autopilotpattern/consul:1.0.0r43`.

Reply:
This file has been neglected since I've been iterating on `examples/compose/local-compose.yml` while figuring stuff out.

I've actually been working against a locally built `1.0.0r47` that incorporates autopilotpattern/consul#46 and autopilotpattern/consul#47 in an effort to make this multi-DC capable from the start.

I will definitely revisit this file once `local-compose.yml` stabilizes